Dockerfile.kubetest2

```diff
@@ -17,7 +17,7 @@ COPY kubetest2/ .
 RUN go install ./...
 WORKDIR $GOPATH/github.com/aws/aws-k8s-tester/e2e2
 COPY e2e2/ .
-RUN go test -c ./test/cases/nvidia -o $GOPATH/bin/e2e-nvidia
+RUN go test -c ./test/cases/nvidia -o $GOPATH/bin/e2e-nvidia && go test -c ./test/cases/netpol -o $GOPATH/bin/e2e-netpol
```

Comment:
Can you move this to a separate `RUN` for readability? We're not too concerned about minimizing the number of layers in this image